src/views/v2/list/event/date-tag.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * View: List View - Single Event Date Tag
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/list/event/date-tag.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+
+$event = $this->get( 'event' );
+?>
+<div class="tribe-events-calendar-list__event-date-tag tribe-common-g-col">
+	<time class="tribe-events-calendar-list__event-date-tag-datetime" datetime="1970-01-01T00:00:00+00:00">
+		<span class="tribe-events-calendar-list__event-date-tag-weekday">Wed</span>
```

Comment:
Maybe if we have some time, given that we already have data for the events, we could use the real date here.

Reply:
Yeah, I don't know the template tags for this data. If you or @bordoni want to go in and fix those up, feel free. I just didn't want to spend a bunch of time looking.

Reply:
Ok, right now we don't have template tags for those specifically. But, yeah, we can take a look at these later.